sdk/core/azure-core-amqp/pom.xml

```diff
@@ -57,7 +57,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.2.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;current} -->
+      <version>1.2.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
Should Azure Core AMQP just peg to current? Do we foresee situations where we would release Azure Core wihtout releasing Azure Core AMQP?

Reply:
Yes, we might release core without having to release core-amqp.